visidata/freeze.py

```diff
@@ -51,7 +51,7 @@ def reload(self):
                     row.append(val)
 
 
-Sheet.addCommand("'", 'freeze-col', 'sheet.addColumnAtCursor(StaticColumn(cursorCol))', 'add a frozen copy of current column with all cells evaluated')
+Sheet.addCommand("'", 'freeze-col', 'sheet.addColumnAtCursor(StaticColumn(cursorCol))', 'Add a frozen copy of current column. This locks the cell values.')
```

Comment:
"Locked" isn't quite right.  It's more like, a completely evaluated copy.  Maybe 'frozen' is the wrong nomenclature too; 'snapshot' maybe?

Reply:
Snapshot feels much better to me.